exec/supervisor/supervisor.conf

```diff
@@ -0,0 +1,10 @@
+processes:
```

Comment:
Shouldn't all these lines be commented out? AFAIR, these represent some default config value, no?

Reply:
Can be, since there is hard to define a default config - everyone has to put its own values here.